db/update-db.py

```diff
@@ -7,7 +7,10 @@
         'CallNumber', 'NumEnrolled', 'MaxSize', 'CourseTitle', 'CourseSubtitle', 
```

Comment:
It's a field specified by CUIT in their courses JSON file. May be relevant when listing details on courses. (Often elaborates on the class, but not always. It might prove to just be a duplicate field in the future.) Should I add a comment clarifying what that list is?